news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
+date: 2017-11-05
+--- 
+
+![](filepath) BANNER IN PROGRESS
+
+After a long hiatus, Chaos, Voli and I are happy to announce the revival of official beatmapping contests: Welcome to the _osu! Mapping Olympiad!_
+
+Let me elaborate on the concept for the osu! Mapping Olympiad. To ensure that there is a healthy and consistent flow of official contests, 
+we have worked on a concept which involved all types of modes - three contests per mode will happen annually in a specific order. 
+To keep things interesting, we have also worked on different kind of contests - so stay tuned in the upcoming months!
+
+### Songs (STILL NEED HELP WITH THE SONG PREVIEWS)
+
+To start off, we are launching this contest with not one, but **three** songs for you to tackle! Showcasing a variety of our Featured Artists, have yourself a listen to the tracks you’ll be taking on:
+
+- Kuba Oms - Beautiful Uncertainty (Redux)
+- OISHII - Honey Snowflakes
+- cySmix - Escapism
+
+There’s going to be something for everyone’s taste - funky rhythms by Kuba Oms, fast-paced goodness by OISHII and some chill beats by cYsmix!
+
+### Ruleset
+
+- This contest is for the **osu!** (standard) mode only.
```

Comment:
omit the standard, osu! is just called osu!

Reply:
I took this from the standard Aspire post but I see that previous posts (MBCs for example) used osu! gamemode instead. Probably best to go with that here as well, now that I've thought about it.

Reply:
Oki